src/providers/data_provider_fo.c

```diff
@@ -594,6 +594,14 @@ static void be_resolve_server_done(struct tevent_req *subreq)
     tevent_req_error(req, ret);
 }
 
+void dump_be_svc_data(struct be_svc_data *svc)
+{
+    DEBUG(SSSDBG_OP_FAILURE, "be_svc_data: name=[%s] last_good_srv=[%s] "
+                             "last_good_port=[%d] last_status_change=[%lld]\n",
+                             svc->name, svc->last_good_srv, svc->last_good_port,
+                             (long long) svc->last_status_change);
```

Comment:
`last_status_change` is `time_t`
Would it make sense to rebase (and use) on top of recently merged 54aabca0b65037600cf5238c452698e81a407bc5?

Reply:
done